Project.toml

```diff
@@ -13,6 +13,7 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 Missings = "e1d29d7a-bbdc-5cf2-9ac0-f12de2c33e28"
 PooledArrays = "2dfb63ee-cc39-5dd5-95bd-886bf059d720"
+PrettyTables = "08abe8d2-0d0c-5749-adfa-8a2ac140af0d"
```

Comment:
`[compat]` entry is missing (we can add it later)

Reply:
Yes, I need to release the 1.0 before adding the `[compat]` here.